test/starlark_tests/targets_under_test/ios/BUILD

```diff
@@ -1023,6 +1023,20 @@ ios_extension(
 
 # ---------------------------------------------------------------------------------------
 
+ios_application(
```

Comment:
i see some other rules have tags "notap" and "manual", not sure if they should be here as well

Reply:
Everything should have `manual`, `notap` is a google thing so probably doesn't matter anymore, but also wouldn't hurt to include.

Reply:
done